bin/p2-rctl/main.go

```diff
@@ -153,6 +153,15 @@ func main() {
 	}
 }
 
+// SessionName returns a node identifier for use when creating Consul sessions.
+func SessionName() string {
+	hostname, err := os.Hostname()
+	if err != nil {
+		hostname = "unknown hostname"
+	}
+	return fmt.Sprintf("p2-rctl:%s:%s", hostname, time.Now().Format("0102-150405"))
```

Comment:
Not useful IMHO. Note that the hostname comes first, and Consul exposes a `CreateIndex` property on the session itself to order by creation (logical) time.